internal/daemonset/daemonset_test.go

```diff
@@ -775,10 +777,11 @@ var _ = Describe("GetNodeLabelFromPod", func() {
 					constants.ModuleNameLabel: moduleName,
 					kernelLabel:               "some kernel",
 				},
+				Namespace: namespace,
 			},
 		}
 		res := dc.GetNodeLabelFromPod(&pod, "module-name")
-		Expect(res).To(Equal(getDriverContainerNodeLabel("module-name")))
+		Expect(res).To(Equal(getDriverContainerNodeLabel(namespace, "module-name")))
```

Comment:
I realize that this change replicates what we were previously doing, but we're never actually verifying that `getDriverContainerNodeLabel` or `getDevicePluginNodeLabel` do things correctly.
```suggestion
		Expect(res).To(Equal("kmm.node.kubernetes.io/some-namespace.module-name.ready"))
```

Reply:
If they fails then the test for `GetNodeLabelFromPod` will fail as well. Did I misunderstood your comment?